packages/bpk-component-bottom-sheet/src/BpkBottomSheetInner.tsx

```diff
@@ -0,0 +1,136 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { type ReactNode } from "react";
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import CSSTransition from 'react-transition-group/CSSTransition';
+
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import { BpkButtonLink } from '../../bpk-component-link';
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import BpkCloseButton from '../../bpk-component-close-button';
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import BpkNavigationBar from '../../bpk-component-navigation-bar';
+import { cssModules } from "../../bpk-react-utils";
+
+import STYLES from './BpkBottomSheetInner.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+type Props = {
+  actionText?: string;
+  children: ReactNode;
+  className?: string;
+  closeLabel?: string;
+  contentClassName?: string;
+  id: string;
+  onAction?: () => void;
+  onClose: () => void;
+  title?: string;
+  wide?: boolean;
+  exiting: boolean;
+}
+
+const BpkBottomSheetInner = ({
+  actionText = '',
+  children,
+  className = '',
+  closeLabel = '',
+  contentClassName = '',
+  exiting,
+  id,
+  onAction = () => null,
+  onClose,
+  title = '',
+  wide = false
+}: Props) => {
+  const classNames = [getClassName('bpk-bottom-sheet')];
+
+  if (wide) {
+    classNames.push(getClassName('bpk-bottom-sheet--wide'))
+  }
+  if (className) {
+    classNames.push(className);
+  }
```

Comment:
This can be simplified to:

```suggestion
  const classNames = getClassName(
     'bpk-bottom-sheet',
     wide && 'bpk-bottom-sheet--wide',
     className
  );
```

Then where you use this variable you can remove the `.join(' ')`